src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractViolationReporter.java

```diff
@@ -42,7 +42,7 @@
      *
```

Comment:
`severity level of the messages` => `severity level of the violations`

Reply:
Done